app/Services/AuthServiceProvider.js

```diff
@@ -114,14 +125,14 @@ async function checkPasswordValid(id, password) {
   });
 
   if (!user) {
-    throw new ApiError(httpStatus.NOT_FOUND, 'Account not found');
+    throw new ApiError(httpStatus.NOT_FOUND, 'Account not found', 'not-found');
```

Comment:
I think this should be the `email` because there should be an red error on the desktop which says "Email not found" or something else.

Reply:
This function, that checks if the password is right, is only used by the function that changes the password. Therefore we parse the userId from the JWT. In this function there is no property email

Reply:
ok, sorry. Than skip this.